src/mscorlib/src/System/IO/StreamReader.cs

```diff
@@ -71,6 +71,9 @@ internal static int DefaultBufferSize
         // This is used only for preamble detection
         private int bytePos;
 
+        [NonSerialized]
+        private StringBuilder builder;
```

Comment:
Yeah, but there are other private fields already in here with an underscore, so we already have a mix.